test_scripts/Defects/8_1/3832_HMI_PTU_after_Ignition_cycle.lua

```diff
@@ -0,0 +1,121 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3832
+---------------------------------------------------------------------------------------------------
+-- Description: SDL sends UP_TO_DATE in case of HMI PTU after Ignition cycle
+--
+-- Steps:
+-- 1. Core and HMI are started
+-- 2. In HMI switch on PTU using in-vehicle modem option
+-- 3. Policy Server is running and exchange_after_x_ignition_cycles = 1
+-- 4. App starts registration
+-- SDL does: start PTU and sends SDL.OnStatusUpdate(UPDATE_NEEDED, UPDATING) to HMI
```

Comment:
```suggestion
-- SDL does:
-- - start PTU and send SDL.OnStatusUpdate(UPDATE_NEEDED, UPDATING) to HMI
```
I propose to use common style

Reply:
@VjKlepikov Done in 978fa88